### PR TITLE
Allow fullscreen display selection

### LINKED ISSUE #298: Fullscreen mode seems to always move window to main screen when there are multiple screens
It's reported to happen on Windows and definitely happens on Debian Testing with Awesome WM. It's not a problem on Awesome WM, due to shortcut for moving window between screens, but it may cause trouble on Windows and on other window managers on Linux, so it's worth checking.

To replicate, move Overte window to secondary screen and then switch to full screen mode. The window will move to the primary screen on its own, while the desired behavior would be for it to stay on the screen where it was placed in windowed mode.

### PR DESCRIPTION
**Pull request by daleglass**
This adds an option to the graphics screen to choose which screen to use for full screen mode.

Screenshot_20230106_153003

It adds the API functions:

* Render.getScreens()
* Render.setFullScreenScreen(QString)
* Render.getFullScreenScreen()

Fixes #298

### CODE REVIEW

**Comment by ksuprynowicz** [comment]
It works very well for me on Xorg

**Review by akamicah — APPROVED** [review]

File: libraries/ui/src/ScreenName.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
+//
+//  Created by Dale Glass on 7/01/2023
+//  Copyright 2022 Overte e.V.
```
Comment (JulianGro) [inline libraries/ui/src/ScreenName.cpp:3]:
```suggestion
//  Copyright 2023 Overte e.V.
```

;)

File: libraries/ui/src/ScreenName.h [hunk 2]
```diff
@@ -0,0 +1,32 @@
+//
+//  Created by Dale Glass on 7/01/2023
+//  Copyright 2022 Overte e.V.
```
Comment (JulianGro) [inline libraries/ui/src/ScreenName.h:3]:
```suggestion
//  Copyright 2023 Overte e.V.
```

**Review by JulianGro — CHANGES_REQUESTED** [review]
Would be nice if you updated the copyright headers on all of the changed files.

**Pull request merged**